src/components/List.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import useFirestore from '../hooks/useFirestore';
 import { db } from '../lib/firebase';
 import Error from './Error';
+import calculateEstimate from './../lib/estimates';
+// import { formatDistance, formatDuration, addMilliseconds, toDate, getTime } from 'date-fns';
```

Comment:
These date-fns import statements are for the demo only and will be removed before merging into main.